datafusion/physical-plan/src/sorts/merge.rs

```diff
@@ -97,6 +97,26 @@ pub(crate) struct SortPreservingMergeStream<C: CursorValues> {
     /// Cursors for each input partition. `None` means the input is exhausted
     cursors: Vec<Option<Cursor<C>>>,
 
+    /// Configuration parameter to enable round-robin selection of tied winners of loser tree.
+    enable_round_robin_tie_breaker: bool,
```

Comment:
Do we want to make this configurable? Maybe it makes sense to enable it by default if it's generally faster this way, this will avoid a branch as well or extra code generation.

Reply:
With this, it is easier to test between enabled one and disabled one

Reply:
Why would we need to test the disabled one? If we don't use it, seems no use to test it?

Reply:
We can see the difference in benchmark and test.

Here is the only place that remove this flag helps, so I guess remove the flag doesn't help too much
```rust
if self.enable_round_robin_tie_breaker && cmp_node == 1 {
```

Reply:
I remove it

Reply:
Nice, seems to have cleaned up things a bit